code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -33,7 +33,7 @@
 	if(prob(40))
 		if(ishuman(L))
 			var/mob/living/carbon/human/H = L
-			if(H.dna && !(HAS_TRAIT(H, TRAIT_MUTATEIMMUNE) || HAS_TRAIT(H, TRAIT_MUTATEIMMUNE))) // NSV13 Don't add mutations for species that are trait immune, IPCs
+			if(H.dna && !(HAS_TRAIT(H, TRAIT_RADIMMUNE) || HAS_TRAIT(H, TRAIT_MUTATEIMMUNE))) // NSV13 Don't add mutations for species that are trait immune, IPCs
```

Comment:
Shouldn't it be like this to check correctly?
```suggestion
			if(H.dna && !(HAS_TRAIT(H, TRAIT_RADIMMUNE) && !HAS_TRAIT(H, TRAIT_MUTATEIMMUNE))) // NSV13 Don't add mutations for species that are trait immune, IPCs
```

Reply:
Oh yeah nevermind me, overlooked the has_trait checks being in one big negated clause and not just being or-linked. Should be all good then.